graphql.gemspec

```diff
@@ -41,6 +41,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency "sequel"
   s.add_development_dependency "sqlite3"
 
+  # TODO(xuorig) probably can in house it:
+  s.add_runtime_dependency "activesupport"
```

Comment:
You could also consider adding this gem as a dependency mbj/inflecto is way smaller that active-support, and used by many other projects.